### DISCUSSION TITLE
Can you give me an example to make sure it's working?

### DISCUSSION THREAD

**Discussion by linrongbin16** [crate-ci/typos#949] [Q&A] [answered]
Hi,

I am trying to use typos in my github actions, as a lint to check word spell.

But I tried to make some typo, for example I write `Thissss is a fffile` in README.md. And write `let thisssss_is_a_fillllle=1;` in main.rs.

Then run below commands:

```bash
typos
```

It seems nothing happened, would you please share knowledge?

**Comment by epage** [comment, marked as answer]
Typos is not a typical editor spell checker that has a list of known words and tries to coerce your test to that list of known words.  Instead, it works off of a hard coded list of common typos, correcting them.  This is done to keep the false-positive rate low.

For test cases, see the source file we build from: https://github.com/crate-ci/typos/blob/master/crates/typos-dict/assets/words.csv

**Reply by linrongbin16** [reply]
thanks for reply!
